libs/common/src/lib/topbar/topbar.component.html

```diff
@@ -16,7 +16,7 @@
     <img
       class="topbar-logo-collapsed"
       [src]="!standalone ? 'assets/logos/nuclia-symbol.svg' : 'assets/logos/nucliadb-symbol.svg'"
```

Comment:
maybe at some point this logo can be deleted,
it was a variant intended for small/mobile resolutions

Reply:
definitely